src/main.ts

```diff
@@ -80,7 +80,13 @@ export class UpgradeCommand implements Command {
 	};
 
 	async runUpgrade(
-		{ version, transforms = [], dependencies: { add = [], remove = [] } = {}, run, postTransform }: VersionConfig,
+		{
+			version,
+			transforms = [],
+			dependencies: { add = [], remove = [] } = {} as Dependencies,
```

Comment:
casting `{}` to `Dependencies` here. TypeScript complained because `{}` was missing an unused, required field